README.md

```diff
@@ -1,36 +1,43 @@
-This module will create an S3 bucket and an IAM user policy based off of the `var.access` variable (either readwrite or readonly). The created policy can then be attached to an externally created IAM user.
+This module will create an S3 bucket and associated IAM policies. The created policies can then be attached to an externally created IAM user.
+
+Bucket versioning and lifecycle rules have been separated into independent sections (IA, Glacier, expire objects) for different app needs.
 
 ## Inputs
 
-| Name                               | Description                                                                                                                                                                                                                                                                  |  Type  |        Default       | Required |
```

Comment:
It's really hard to see what's changed here because of all the formatting changes. In the future, I'd suggest doing a commit with just formatting changes and then a commit with your changes.